function addtwonums(a , b){
    try{
        let sum = a + b;
           return sum;

    }catch(error){
        console.log(error)
    }
}


console.log(addtwonums(2, 5))